### DISCUSSION TITLE
Brightness control?

### DISCUSSION THREAD

**Discussion by DrTautology** [russhughes/st7789_mpy#130] [Q&A]
Is there anyway to control the brightness of the display?

**Comment by russhughes** [comment]
It depends on the display. If the backlight is connected to a PWM-capable pin, you can use it to control the display's brightness.

**Reply by DrTautology** [reply]
Okay, I'm using a pico w and the waveshare 2 inch lcd. I have tried defining the backlight pin as gpio 9, but I get no output from the display when doing this. I could find any examples in your repo for controlling the backlight.
